ci/prometheus-down.yml

```diff
@@ -0,0 +1,12 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: 18fgsa/concourse-task
+
+inputs:
+- {name: prometheus-config}
```

Comment:
The curly braces don't do anything here, but cool if it makes this more readable.

Reply:
Again, cut/paste.  I'm just using what others have done!